platform/platform-migratable/src/main/java/org/codice/ddf/platform/migratable/impl/PlatformMigratable.java

```diff
@@ -48,6 +48,10 @@
 
     private static final Path USERS_PROPERTIES = Paths.get("etc", "users.properties");
 
+    private static final Path DDF_METACARD_ATTRIBUTE_RULESET = Paths.get("etc/pdp", "ddf-metacard-attribute-ruleset.cfg");
```

Comment:
If you're using Paths, the correct way to do this would be Paths.get("etc", "pdp", ...) and lose the "/" in the first one.